pkg/webhook/risingwave_conversion.go

```diff
@@ -83,8 +83,6 @@ func ConvertStorages(obj *v1alpha1.RisingWave) {
 		stateStore.S3.RisingWaveS3Credentials.SecretName = stateStore.S3.Secret
 	case stateStore.GCS != nil && stateStore.GCS.Secret != "":
 		stateStore.GCS.RisingWaveGCSCredentials.SecretName = stateStore.GCS.Secret
-	case stateStore.AliyunOSS != nil && stateStore.AliyunOSS.Secret != "":
-		stateStore.AliyunOSS.RisingWaveAzureBlobCredentials.SecretName = stateStore.AliyunOSS.Secret
```

Comment:
Why is this deleted?

Reply:
in the old definition, we define the `Secret` and `Credentials`, but i find we already shift to `Credentials`, and will deprecate the old one `Secret` in future. So I didn't create `Secret` in this definition. Then we do not have `Secret` and also don't need this step.